ics/event.py

```diff
@@ -71,6 +72,7 @@ def __init__(self,
             alarms (:class:`ics.alarm.Alarm`)
             categories (set of string)
             status (string)
+            classification (string)
```

Comment:
Is it defined as `classification` or `class` in the RFC?

Reply:
Maybe clashes with python property `class`?

Reply:
OK, more like a syntax error:

    >>> x.class
      File "<stdin>", line 1
        x.class
              ^
    SyntaxError: invalid syntax

Reply:
In the RFC they call it `classification` and the name of the Property is `CLASS`, so they use both names as you can see here https://tools.ietf.org/html/rfc5545#section-3.8.1.3

when did you get the syntax error?

Reply:
Ok. I meant using `class` instead of `classification` would not work.

Reply:
Okay, now in understand.
Hope everything is fine now